ToDoWebApi/Controller/UserController.cs

```diff
@@ -67,9 +67,11 @@ public IActionResult AddUser([FromBody] CreateUserViewModel newUser)
     [HttpPut("{id}")]
     public IActionResult UpdateUser([FromBody] UpdateUserViewModel updateUser, int id)
     {
-        UpdateUserCommand command = new UpdateUserCommand(_context, id);
+        UpdateUserCommand command = new UpdateUserCommand(_context);
         command.Model.Id = id;
-        command.Model = updateUser;
+        command.Model.Email = updateUser.Email;
+        command.Model.Password = updateUser.Password;
+        command.Model.NewPassword = updateUser.NewPassword;
```

Comment:
sırf bu command'in controller'ında model oluşturuyorsun diye bunları tek tek yazıyorsun. Constructor'da model oluşturmana ne gerek var? Hayır command'indeki modelin classı ve buradaki modelin classı farklı olsa mapliyorsun derim ama modeller aynı. O zaman hiçbir amacı yokken command'de model oluşturmak yerine neden buradaki modeli command'inin modeline setlemiyorsun?